persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -18,7 +18,8 @@ CREATE TABLE Conversation (
     last_notified_message_date TEXT,
     last_read_date TEXT DEFAULT "1970-01-01T00:00:00.000Z" NOT NULL,
     access_list TEXT AS List<ConversationEntity.Access> NOT NULL,
-    access_role_list TEXT AS List<ConversationEntity.AccessRole> NOT NULL
+    access_role_list TEXT AS List<ConversationEntity.AccessRole> NOT NULL,
+    keying_material_last_update INTEGER DEFAULT 0 NOT NULL
```

Comment:
```suggestion
    mls_last_keying_material_update INTEGER DEFAULT 0 NOT NULL
```

Sorry, didn't see this earlier but the naming is a bit inconsistent with other timestamps, also I've been prefixing all mls related fields with `mls_`

Maybe also move it to the other mls fields.